great-idea-website/css/index.css

```diff
@@ -63,4 +64,154 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+header {
+	display: inline-block;
+	width: 600px;
+	background-color: lightgrey;
+	width: 100%;
+	height: 50px;
+	position: fixed;
+}
+
+li {
+	display: inline-block;
+	padding: 0 10px;
+}
+
+nav {
+	display: inline-block;
+	width: 999px;
+	margin-top: 15px;
+	padding-right: 25px;
+	display: inline-block;
+	margin-left: 300px;
+}
+a {
+	border: 2px solid black;
+	background-color: lightblue;
+	text-decoration: none;
+	color: white;
+	display: inline-block;
+	width: 80px;
+	height: 25px;
+	padding: 3px;
+	padding-left:10px;
+}
+
+a:hover {
+	background-color: yellow;
+	color: black;
+}
+.logo {
+	display: inline-block;
+	height: 30px;
+	width: 100px;
+
+	
+}
+
+
+
+.container {
+	width: 876px;
+	margin: 0 auto;
+}
+.sectionOne {
+	display: inline-block;
+}
+
+
+
+.iod {
+	width: 45%;
+	height: 300;
+	
+	padding: 45px;
+	font-size: 60px;
+	display: inline-block;
+	
+}
+.button1 {
```

Comment:
The style that make buttons _feel_ most like buttons is 'cursor: pointer;' - it makes them look clickable! 😁